android/app/buildConfigs.gradle

```diff
@@ -0,0 +1,55 @@
+// See https://github.com/luggit/react-native-config/blob/master/android/dotenv.gradle for reference
+
+def createConfig () {
+    def buildConfig = [:]
+
+    if (!System.env["BUILD_CONFIG_NAME"]) {
+        println("*****************++++++++*********")
+        println("*** Missing BUILD_CONFIG_NAME ****")
+        println("************************++++++++**")
+
+        // Set to empty array to prevent failing for other tasks
+        project.ext.set("buildConfig", buildConfig)
+
+        return
+    }
+
+    def buildConfigName = System.env['BUILD_CONFIG_NAME']
+    println("Using build config $buildConfigName")
+
+    def command = "yarn babel-node tools/create-xcconfigs config $buildConfigName --directory android"
+    def proc = command.execute()
+    proc.waitFor()
+
+    if (proc.exitValue() != 0) {
+        println("Failed to create xcconfig file: $proc.err.text")
+        return
+    }
+    println("Created xcconfg file: $proc.in.text")
+
+    def buildConfigProperties = new Properties()
+    buildConfigProperties.load(new FileInputStream("$project.rootDir/buildConfig.tmp.xcconfig"))
```

Comment:
The FileInputStream should be closed afterwards, also maybe exceptions need to be handled

Reply:
I don't see a point in handling exceptions right now, shouldn't happen because it is perfect code :stuck_out_tongue: . Also, if an exception occurs, we have to investigate why it fails anyway. Closing the input stream though.